awx/main/dispatch/pool.py

```diff
@@ -320,15 +324,14 @@ def __init__(self, *args, **kwargs):
         super(AutoscalePool, self).__init__(*args, **kwargs)
 
         if self.max_workers is None:
-            settings_absmem = getattr(settings, 'SYSTEM_TASK_ABS_MEM', None)
-            if settings_absmem is not None:
-                # There are 1073741824 bytes in a gigabyte. Convert bytes to gigabytes by dividing by 2**30
-                total_memory_gb = convert_mem_str_to_bytes(settings_absmem) // 2**30
-            else:
-                total_memory_gb = (psutil.virtual_memory().total >> 30) + 1  # noqa: round up
```

Comment:
to give some history, the use of `psutil` here is completely ancient as of the original introduction https://github.com/ansible/awx/pull/2266

Then https://github.com/ansible/awx/pull/10954 changed to use the ansible-runner memory util. The issue of max_workers parity may not have really been on the radar at that point.

@kdelee did some work to fix basically this same problem in https://github.com/ansible/awx/pull/11800, but didn't touch the fact that psutil was used in a now one-off way.